code/media/src/main/java/com/adobe/marketing/mobile/media/internal/MediaOfflineService.java

```diff
@@ -295,7 +295,7 @@ boolean reportCompletedSessions() {
                 return false;
             }
 
-            if (url == null || url.length() == 0) {
+            if (url == null || url.isEmpty()) {
```

Comment:
nit: You can also use `StringUtils.isNullOrEmpty()` from Core. Not particular about it though.

Reply:
Updated